KarateTraining/src/test/java/signuptask/signup.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Testing the react herokuapp for Sign up users
+
+  Background:
+    * url baseUrl
+
+  @postNewUser
+  Scenario: Sign up a new user in the app
+
+    * def func = karate.call('classpath:src/test/RandomStringGenerator.js', [6])
+    * def randomEmail = func + '@gmail.com'
+    * def randomPassword = karate.call('classpath:src/test/RandomStringGenerator.js', [6])
+    * def randomUsername = karate.call('classpath:src/test/RandomStringGenerator.js', [6])
+
+    * def user = { "user": { "email": '#(randomEmail)', "password":'#(randomPassword)', "username": '#(randomUsername)'}}
+
+
+    Given path 'users'
+    And request user
+    When method post
+    Then status 200
+    Then match response == read('classpath:signInSignUpResponse.json')
+    And assert response.user['email'] == user['user']['email']
+    And assert response.user['username'] == user['user']['username']
+    * def signIn = call read('classpath:src/test/java/signintask/signin.feature@postSignInUser')
```

Comment:
Add this call like an `And` instruction because is a step of this scenario. Something like this:

`And def siginUn = call... `